native/src/components/RemoteContent.tsx

```diff
@@ -99,7 +99,7 @@ const RemoteContent = (props: RemoteContentProps): ReactElement | null => {
         log(message.message, 'warning')
       } else {
         const error = new Error(
-          message.message ? JSON.stringify(message.message) : 'Unknown message received from webview',
+          message.message !== undefined ? JSON.stringify(message.message) : 'Unknown message received from webview',
```

Comment:
💭  Hmmm, if the error message is an empty string, I think I'd find the `Unknown message received from webview` error clearer than the empty string. Though I'm not sure how that would ever happen either.